### PR TITLE
#457 - Theme settings only override settings.php.

### PR DESCRIPTION
**Pull request by julien-**
### Description

Now if `$theme_debug` or `$allow_css_double_underscores` is defined in _settings.php_ or _settings.local.php_ it will be taken in consideration.

### CODE REVIEW

File: atomium/includes/config.inc [hunk 1]
```diff
@@ -2,9 +2,8 @@
 
 /**
  * @file
- * config.inc
+ * Config file.
```
Comment (drupol) [inline atomium/includes/config.inc]: Unrelated changes.

**Review by drupol — CHANGES_REQUESTED** [review]
Small stuff to change before approving

File: atomium/includes/config.inc (hunk 1 above)
Comment (aritomelo) [reply atomium/includes/config.inc]: @drupol ++

**Comment by drupol** [comment]
@julien- Could you update this PR so I can merge and tag release tonight ?

**Comment by julien-** [comment]
@drupol It's Done
Thanks again

**Pull request merged**